ui-components/src/main/java/org/dhis2/ui/dialogs/bottomsheet/BottomSheetDialog.kt

```diff
@@ -71,6 +72,7 @@ class BottomSheetDialog(
             setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
             setContent {
                 DHIS2Theme {
+                    val scrollState = rememberLazyListState()
```

Comment:
This dialog is also used for the login pop up and the validation pop up in the form, maybe its worth directly calling the bottomSheetShell from the period selector to avoid possible unwanted effects in these other usecases?

Reply:
It does not affect the login nor any other usage it could have. The scrollState is shared with the content composable in order to allow adding a LazyColumn.